css/style.css

```diff
@@ -231,3 +231,97 @@ footer h5 {
 .p3-text p {
   font-style: italic;
 }
+
+.main2 {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.section2 {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  gap: 30px;
+}
+
+.section2 h1 {
+  color: var(--orange);
+}
+
+label {
+  font-size: 12px;
+  font-weight: 400;
+  color: var(--grey-dark);
+}
+
+.fieldset {
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  flex-wrap: wrap;
+  border: none;
+  gap: 30px;
+}
+
+.what {
```

Comment:
Try to name your classes with a bit more meaning eg `.labelInputWrapper` or something